src/components/portfolio/VolatilityLineEntry.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Tooltip } from '@material-ui/core';
+
+const useStyles = makeStyles({
+  line: {
+    position: 'absolute',
+    height: '2rem',
+    width: '0.25rem',
+    transform: 'translate(0%, -50%)',
+  },
+  tooltip: {
+    fontSize: '0.8rem',
+    fontWeight: 600,
+    lineHeight: '1.2rem',
+    whiteSpace: 'pre-line',
+  },
+});
+
+type VolatilityLineEntryProps = {
+  volatilityValue: number;
+  tooltipText: string;
+  color: string;
+};
+
+const VolatilityLineEntry: React.FC<VolatilityLineEntryProps> = ({
+  volatilityValue,
+  tooltipText,
+  color,
+}) => {
+  const classes = useStyles();
+
+  return (
+    <Tooltip className={classes.tooltip} title={tooltipText}>
```

Comment:
If multiple stocks can have the same volatility and you want to display them in the same tooltip (like I did in analysts) then the way it was before (as weird as it may have been) was actually correct. The problem lies with material-ui overriding the whiteSpace attribute and for some reason jss doesnt allow !important to be added to whiteSpace. However, if only ever one name is displayed in the same tooltip then it is fine the way it is now.